src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -1585,6 +1586,11 @@ enum VectorMask {
   INSN(vfadd_vf,  0b1010111, 0b101, 0b000000);
   INSN(vfrsub_vf, 0b1010111, 0b101, 0b100111);
 
+  // Vector Floating-Point Sign-Injection Instructions
+  INSN(vfsgnj_vf, 0b1010111, 0b101, 0b001000);
+  INSN(vfsgnjx_vf, 0b1010111, 0b101, 0b001010);
+  INSN(vfsgnjn_vf, 0b1010111, 0b101, 0b001001);
```

Comment:
Not used anywhere?

Reply:
I can remove them if you think it's better to do it.
Reason I added them is that they're quite similar instructions and it's annoying to lookup the instruction formatting in spec and add every single one when needing them.

Reply:
Ah, I would suggest remove them for test coverity reasons. We can add them back when needed.